index.js

```diff
@@ -25,6 +26,13 @@ module.exports = {
       destDir: '/pretender',
     });
 
+    pretenderTree = concat(pretenderTree, {
+      outputFile: '/pretender/pretender.js',
+      footer: 'define(\'pretender\', [], function() {' +
+              '  return self.Pretender;' +
+              '});',
+    });
```

Comment:
For the full context of why I'm doing this please see the comment in the PR
> output umd
https://github.com/pretenderjs/pretender/pull/248#issuecomment-462678871